src/app/shared/search-box/query-item/query-item.component.html

```diff
@@ -8,8 +8,8 @@
 
   <span [class.font-weight-bold]="isCollectionItem()"
         [title]="queryItem.text"
-        class="ml-1 text-truncate">
-    {{queryItem.text}}
+        class="ml-1">
+    {{queryItem.text | truncate:30}}
```

Comment:
Why are we changing it back?

Reply:
Because of long collection names. For my opinion 30 characters is enough...

Reply:
Oh, I see. We changed it to `text-truncate` but forgot to set the maximum width.